src/main/java/org/opensearch/security/dlic/rest/api/SecurityConfigAction.java

```diff
@@ -43,18 +43,19 @@
 import org.opensearch.threadpool.ThreadPool;
 
 import com.fasterxml.jackson.databind.JsonNode;
+import static org.opensearch.security.dlic.rest.support.Utils.addRoutesPrefix;
 
 public class SecurityConfigAction extends PatchableResourceApiAction {
 
-    private static final List<Route> getRoutes = Collections.singletonList(
-            new Route(Method.GET, "/_opendistro/_security/api/securityconfig/")
-    );
+    private static final List<Route> getRoutes = addRoutesPrefix(Collections.singletonList(
+            new Route(Method.GET, "/securityconfig/")
+    ));
 
-    private static final List<Route> allRoutes = new ImmutableList.Builder<Route>()
+    private static final List<Route> allRoutes = addRoutesPrefix(new ImmutableList.Builder<Route>()
             .addAll(getRoutes)
```

Comment:
this needs to be added to the final collection, otherwise it adds prefix twice.

Reply:
fixed it in Routes()